### PR TITLE
Allow for 3rd party projects

### PR DESCRIPTION
**Pull request by wblankenship**
Addresses #293

### CODE REVIEW

**Comment by maxrimue** [comment]
@tedgaydos What's your opinion on this? We'll have to manually merge this as this PR contains some outdated code snippets from an older version of n

**Comment by wblankenship** [comment]
Yeah, I created this off of the August 15th commit. I can do the heavy lifting of merging into your current codebase if it will be a burden.

Interested in feedback on the approach.

**Comment by maxrimue** [comment]
@wblankenship You don't have to, I'll be able to do this manually

File: bin/n [hunk 1]
```diff
@@ -4,13 +4,29 @@
 # Setup.
 #
 
-VERSION="2.0.2"
+VERSION="1.3.0"
```
Comment (yorkie) [inline bin/n]: BTW, i guess you are working on an error version of tj/n :(

Comment (maxrimue) [inline bin/n]: @yorkie What do you mean by 'error version'?

Comment (yorkie) [inline bin/n]: Oh, I mean @wblankenship was working on the old version 1.3.0 from the diff.

File: bin/n [hunk 2]
```diff
@@ -4,13 +4,29 @@
...
 N_PREFIX=${N_PREFIX-/usr/local}
 BASE_VERSIONS_DIR=$N_PREFIX/n/versions
 
 #
+# Log <type> <msg>
+#
+
+log() {
+  printf "  \033[36m%10s\033[0m : \033[90m%s\033[0m\n" $1 $2
+}
```
Comment (yorkie) [inline bin/n]: @maxrimue this function seems to be existed in the latest codebase.

File: bin/n [hunk 3]
```diff
@@ -4,13 +4,29 @@
...
+
+#
+# Exit with the given <msg ...>
+#
+
+abort() {
+  printf "\n  \033[31mError: $@\033[0m\n\n" && exit 1
+}
```
Comment (yorkie) [inline bin/n]: ditto, so it's weird diff, I didn't track the commits history of @wblankenship, just guess his master branch might not be updated from `tj/n:master` :)

**Comment by yorkie** [comment]
Sorry, I'm just the wrong guy, just ignore my comments :)

**Comment by maxrimue** [comment]
@yorkie You're pointing out the right things, this PR is based on an older version of n, which is why it's trying to add things we removed a while ago and so on, when I merge this manually, however, I can make it fit into the current version again

**Comment by yorkie** [comment]
Yea, looking for this feature would get merged into master :)

**Comment by wblankenship** [comment]
:heart:

**Pull request merged**